### PR TITLE
657-config-details-switch-tags-to-sidebar-like-the-topology-details

### LINKED ISSUE #657: Config Details - Switch tags to sidebar like the topology details

### PR DESCRIPTION
**Pull request by mainawycliffe**
Closes #657

### CODE REVIEW

**Comment by mainawycliffe** [comment]
@moshloop is this what you had in mind?
image

File: src/components/ConfigSidebar/ConfigDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                label: key,
+                value
+              }))
+          : [])
+      ];
+    }
+  }, [configDetails]);
+
+  return (
+    <div className="flex flex-col space-y-2 max-w-full">
```
Comment (moshloop) [inline src/components/ConfigSidebar/ConfigDetails.tsx:40]: Can we switch this to a collapsible panel ?

Comment (mainawycliffe) [reply src/components/ConfigSidebar/ConfigDetails.tsx:40]: Sure

File: src/components/DescriptionCard/index.tsx [hunk 2]
```diff
@@ -7,18 +7,18 @@ type DescriptionCardProps = {
...
+    <div className="flex flex-col max-w-full">
+      <table className="table-auto shadow-none text-sm w-full max-w-full">
         <tbody>
           {items.map((item, index) => (
             <tr
               className="border-slate-100 border-solid border-b border-t p-1"
               key={item.label ?? "" + index}
             >
-              <th className="text-gray-700 font-light  uppercase text-left overflow-auto bg-zinc-50 pl-2">
+              <th className="text-gray-700 font-light  uppercase text-left break-all bg-zinc-50 pl-2">
```
Comment (moshloop) [inline src/components/DescriptionCard/index.tsx:18]: We don't want to break the labels - if anything we want to group by prefixes (labels split with a `/` or `:`)

File: src/components/ConfigSidebar/ConfigDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    isLoading,
+    error
+  } = useGetConfigByIdQuery(configId);
+
+  const displayDetails = useMemo(() => {
+    if (configDetails) {
+      return [
+        {
+          label: "Name",
+          value: configDetails.name
```
Comment (moshloop) [inline src/components/ConfigSidebar/ConfigDetails.tsx:25]: Add in type, created and last updated fields

File: src/components/ConfigSidebar/ConfigDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+              }))
+          : [])
+      ];
+    }
+  }, [configDetails]);
+
+  return (
+    <div className="flex flex-col space-y-2 max-w-full">
+      <div className="flex py-2 flex-row flex-1 items-center border-b border-dashed border-gray-200">
+        <Title title="Tags" icon={<FaTags className="w-6 h-auto" />} />
```
Comment (moshloop) [inline src/components/ConfigSidebar/ConfigDetails.tsx:42]: Use details same as topology

**Pull request merged**